internal/repository/postgresdb/appreciation.go

```diff
@@ -112,33 +116,37 @@ func (appr *appreciationsStore) GetAppreciationById(ctx context.Context, tx repo
 		ToSql()
 
 	if err != nil {
-		logger.Errorf("error in generating squirrel query, err: %v", err)
+		logger.Errorf(ctx,"error in generating squirrel query, err: %v", err)
 		return repository.AppreciationResponse{}, apperrors.InternalServer
 	}
 
+	logger.Debug(ctx," appr: query: ",query)
+	logger.Debug(ctx," appr: args: ",args)
```

Comment:
don't log query and parameter separately.

Reply:
resolved